commands/auth.go

```diff
@@ -91,14 +93,16 @@ func checkValues(authURL, clientID string) error {
 
 func runAuth(cmd *cobra.Command, args []string) error {
 	if grant == "implicit" {
-		return authImplicit()
+		return authImplicit("token")
```

Comment:
Maybe a switch here will be easier to read?

Reply:
I prefer if statements for smaller blogs, but I get the point. I think there's a sensor that fires off for developers if they see > 2 if statements that says: this must be a switch statement.